blocks/default-output-block/output-types/default.jsx

```diff
@@ -46,7 +46,7 @@ const chartBeatCode = (accountId, domain) => {
 
 const querylyCode = (querylyId, querylyOrg, pageType) => {
   const querylyInit = `
-    queryly.init("${querylyId}", document.querySelectorAll("main, .header-fix-nav"));
+    queryly.init("${querylyId}", document.querySelectorAll("#fusion-app"));
```

Comment:
This will add queryly to every page with the default output type even if the client doesn't use the queryly integration. Is that the desired behavior?

Reply:
@beltrancaliz I don't see that happening – I only see the queryly script on websites that have **querylyId** set in blocks.json. 

Example – **querylyId** is set on The Gazette: 
https://github.com/WPMedia/Fusion-News-Theme/blob/master/blocks.json#L265
And I see the queryly script when I look in the Network tab on that site: 
https://corecomponents.arcpublishing.com/pf/all-blocks/?_website=the-gazette
<img width="1518" alt="Screen Shot 2021-01-04 at 1 49 00 PM" src="https://user-images.githubusercontent.com/3799948/103568738-15a20c80-4e94-11eb-80eb-bcdf04f2434f.png">


But **querylyId** is _not_ set on The Prophet: 
https://github.com/WPMedia/Fusion-News-Theme/blob/master/blocks.json#L200
And I do not see the script loading on that page: 
https://corecomponents.arcpublishing.com/pf/all-blocks/?_website=the-prophet
<img width="1539" alt="Screen Shot 2021-01-04 at 1 49 07 PM" src="https://user-images.githubusercontent.com/3799948/103568827-408c6080-4e94-11eb-978b-2c90545b4b17.png">

Reply:
@scarothers got it, I'm assuming the selector update it will make it behave the same way